haiku-apps/toner/toner-1.0.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A tone generator"
+DESCRIPTION="Toner, a tone generator for aligning levels in your playback \
+system."
+HOMEPAGE="https://github.com/HaikuArchives/Toner/"
```

Comment:
```diff
-HOMEPAGE="https://github.com/HaikuArchives/Toner/"
+HOMEPAGE="https://github.com/HaikuArchives/Toner"
```
(trailing slash no needed here)

Everything else looks good to me too. (But I did not test anything yet.)
Oops:
```diff
-COPYRIGHT="2001 by Ben Loftis"
+COPYRIGHT="2001 Ben Loftis"
```

Reply:
@fbrosson for refference, when do they need to use the traling slash, and when not?

Reply:
For URIs with no path, ie. http://example.org/ the trailing slash _is_ required.
Some (most?) browsers hide the trailing slash, but this does not mean a URI without it is valid. As a matter of fact it is _not_ valid,

Now, for all other URIs, the rule is simple. When you type e.g.https://github.com/HaikuArchives/Toner in your browser's address bar, it will add a / if it is needed, or it will leave that unchanged if it is not needed. (That's the case for all project repos in GitHub.)
You can see this with curl by displaying the header of the reply to a request:
```
curl -I https://github.com/HaikuArchives/Toner
```
It does not send you a redirect.

On sites for which you should include the `/`, if you don't then you get a redirect:
```
curl -I https://www.w3.org/support
```
HTTP/1.1 301 Moved Permanently
Location: https://www.w3.org/support/
```

Reply:
I will make the change when updating COMMIT to the merged commit.